test/java/seedu/addressbook/commands/DeleteCommandTest.java

```diff
@@ -33,14 +33,14 @@
     @Before
     public void setUp() throws Exception {
         Person johnDoe = new Person(new Name("John Doe"), new Phone("61234567", false),
-                new Email("john@doe.com", false), new Address("395C Ben Road", false), new UniqueTagList());
+                new Email("john@doe.com", false), new Address("395C Ben Road", false), new HashSet<>());
```

Comment:
Does the new constructor you introduced help here?

```java
public Person(Name name, Phone phone, Email email, Address address, Tag... tags)
```

Reply:
On second thought, I think you should remove the constructor. AB-4 doesn't have it.

Reply:
The new constructor was added to replace the `UniqueTagList` constructor:
```Java
public UniqueTagList(Tag... tags)
```
which AB4 doesn't have.
Although this is only used in tests, so I wouldn't really mind removing it as well to be consistent with AB4.